### DISCUSSION TITLE
[想法征集]如何在技术社区中进一步推广Apache本土社区的志愿建设

### DISCUSSION THREAD

**Discussion by JesseZhou-1** [alc-beijing/SIG#5] [新媒体运营]
**请大家畅所欲言！**

**Comment by gh-Devin** [comment]
目前群里人已经不少了，如何把大家的积极性调动起来都参与进来，我觉得是目前比较重要的事情

**Reply by WillemJiang** [reply]
非常同意， 现在需要商量一个比较好的有效的方式把大家组织起来。

**Reply by JesseZhou-1** [reply]
嗯嗯是的。耿航老师有没有什么好想法，可以给我们出谋划策一下😄

**Reply by gh-Devin** [reply]
得建立相应的激励机制，比如荣誉、周边奖励等等

**Reply by gh-Devin** [reply]
亦或是组织活动让大家有参与或者学习到东西之类的；目前alc活动还是比较少

**Reply by gh-Devin** [reply]
综合整理下就是建立激励机制：
目前参与机制已经制定好了，但如何提高大家积极性是下一步工作，个人认为可以建议相应的激励机制，比如建立年度或季度评比之星；评比翻译之星，播客达人，分享达人，等等荣誉称号以鼓励志愿者参与度；选送周边奖励如衣服，贴纸等等

**Reply by WillemJiang** [reply]
这个提议是挺好的， 目前我们也在商议ALC Beijing管理晋升体系，欢迎耿航老师参与商议。

**Reply by gh-Devin** [reply]
> 这个提议是挺好的， 目前我们也在商议ALC Beijing管理晋升体系，欢迎耿航老师参与商议。

可以啊，非常感兴趣

**Comment by WillemJiang** [comment]
@JesseZhou-1 可以把咱们的ALC Beijing的社区发展计划的草稿放到wiki里面让大家讨论起来。

**Comment by JesseZhou-1** [comment]
OK

Willem Jiang ***@***.***> 于2021年4月14日周三 下午7:56写道：
